### ISSUE TITLE
option -V always gives 1.0.2

### ISSUE THREAD

**Issue by carlosen14** [vishwanatharondekar/gitlab-cli#73]
You have been updated the package but haven't updated the version in code.. if don't want to manage this, read the `package.json` it is a `JSON` and print the `version` field.

if you do 
```bash
$ npx -q git-lab-cli -V
1.0.2
$ npx -q git-lab-cli@1.0.2 -V
1.0.2
$ npx -q git-lab-cli@2.0.0 -V
1.0.2
$ npx -q git-lab-cli@2.0.3 -V
1.0.2
```

**Comment by carlosen14** [comment]
is hardcoded [here](https://github.com/vishwanatharondekar/gitlab-cli/blob/master/index.js#L482)

**Comment by vishwanatharondekar** [comment]
Merged https://github.com/vishwanatharondekar/gitlab-cli/pull/74 Fixes this.

**Comment by carlosen14** [comment]
to be clear.. i didn't tested it, i think wasn't necessary.. hope you do it

**Comment by vishwanatharondekar** [comment]
I made the same changes after seeing your issue. Tested it and then saw that you created MR with same changes. All good. Thanks for MR.

**Comment by carlosen14** [comment]
fixed with #74 . Your welcome, i was stuck for these few problems.
